harmonizer/js/index.mjs

```diff
@@ -0,0 +1,2 @@
+export { composeAndValidate } from "@apollo/federation";
+export { parse as parseGraphqlDocument } from "graphql";
```

Comment:
I'm going to stick with this for now, particularly because it's an internal API and I have difficult to reconcile reservations about snake-case with that pattern, even if `graphql` has decided it's okay themselves, particularly in a world where libraries like Rust's `serde` need to be able to reliably round-trip transformations (between camelcase and snake-case, for example) of property keys.  In my m ind, the obstacles that `GraphQL[A-Z]` produces are not nearly as egregious as the hurdle that `XMLHTTPRequest` would provide, but the results are just as disappointing.